lib/email/receiver.rb

```diff
@@ -1528,7 +1514,7 @@ def add_other_addresses(post, sender, mail_object)
             address_field.decoded
             email = address_field.address.downcase
             display_name = address_field.display_name.try(:to_s)
-            next unless email["@"]
+            next if !email.include?("@")
```

Comment:
While I enjoy writing less code, I find the `.include?` version to be slightly clearer for everyone.